src/lter/bonanza/processing_kernels.R

```diff
@@ -24,7 +24,7 @@ process_0_142 <- function(set_details, network, domain){
     return()
 }
 
-#stream_temperature: STATUS=READY
+#stream_chemistry: STATUS=PENDING
```

Comment:
@spencerrhea what's up with these two kernels that have been set to PENDING?

Reply:
This was a mistake on my part, meant to change this back. Thanks!